config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
+  get '/signup', to: 'users#new'
   root 'static_pages#home'
-  get 'static_pages/home'
-  get 'static_pages/help'
-  get 'static_pages/about'
+  get '/help', to: 'static_pages#help', as: 'helf'
```

Comment:
`as: 'helf'` vậy hiểu cái này để làm gì k e?

Reply:
`to: 'static_pages#help'` vậy hiểu cái này để làm gì k e?

Reply:
e dùng as để đặt tên cho route thành helf
thay vì dùng help_path, help_url thì giờ chuyển thành helf_path và helf_url
thường đặt tên để route dễ nhớ
to: 'static_pages#help' dùng để xác định action mà url /help hướng tới
trong trường hợp này thì sẽ trỏ đến function help ở trong StaticPagesController